src/core/load-fragment.c

```diff
@@ -2512,6 +2512,8 @@ int config_parse_unit_condition_null(
         assert(rvalue);
         assert(data);
 
+        log_syntax(unit, LOG_WARNING, filename, line, 0, "ConditionNull= is deprecated, please do not use.");
```

Comment:
Please use `%s` with `lvalue` instead of `ConditionNULL=`, as this is also used for `AssertNull=`.

Reply:
indeed!